x-pack/plugins/secops/public/store/local/network/reducer.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { reducerWithInitialState } from 'typescript-fsa-reducers';
+
+import {
+  applyNetworkFilterQuery,
+  setNetworkFilterQueryDraft,
+  updateTopDestinationLimit,
+  updateTopSourceLimit,
+} from './actions';
+import { NetworkModel } from './model';
+
+export type NetworkState = NetworkModel;
+
+export const DEFAULT_LIMIT = 10;
```

Comment:
Probably both hosts and this one should have `DEFAULT_LIMIT` moved higher so they both share the same one unless we think they're going to be different at some point.